.github/workflows/verify-dashboards-alerts.yaml

```diff
@@ -0,0 +1,385 @@
+      
+      - name: Get the installation ID
+        run: |
+          RESPONSE=$(curl -L \
+            -H "Authorization: Bearer ${{ env.JWT_TOKEN }}" \
+            https://api.github.com/app/installations)
+
+          echo "INSTALLATION_ID=$(echo $RESPONSE | jq -r .[0].id)" >> $GITHUB_ENV
+      
+      - name: Generate Installation Access Token
+        run: |
+          RESPONSE=$(curl -L \
+            -X POST \
+            -H "Authorization: Bearer ${{ env.JWT_TOKEN }}" \
+            https://api.github.com/app/installations/${{ env.INSTALLATION_ID }}/access_tokens) 
+
+          echo "ACCESS_TOKEN=$(echo $RESPONSE | jq -r .token)" >> $GITHUB_ENV
+
+      - name: Generate Registration Token
+        run: |
+          REPO_OWNER=${{ github.repository_owner }}
+          REPO_NAME=${{ github.event.repository.name }}
+          RESPONSE=$(curl -L \
+            -X POST \
+            -H "Authorization: Bearer ${{ env.ACCESS_TOKEN }}" \
+            https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/actions/runners/registration-token)
+
+          echo "REGISTRATION_TOKEN=$(echo $RESPONSE | jq -r .token)" >> $GITHUB_ENV
+
+      - name: SSH into EC2 Instance and Register Runner
+        env:
+          REGISTRATION_TOKEN: ${{ env.REGISTRATION_TOKEN }}
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y libicu-dev
+
+          # Function to check SSH connectivity
+          function check_ssh {
+              nc -z -w 5 ${{ env.EC2_PUBLIC_IP }} 22
+          }
+
+          echo "Checking SSH connectivity to ${{ env.EC2_PUBLIC_IP }}..."
+
+          # Loop until SSH is available
+          while ! check_ssh; do
+              echo "SSH is not yet available. Retrying in 10 seconds..."
+              sleep 10
+          done
+
+          echo "SSH is now available. Connecting now..."
+
+          ssh -o StrictHostKeyChecking=no -i ${{ secrets.AWS_KEY_NAME }}.pem ubuntu@${{ env.EC2_PUBLIC_IP }} << EOF
+            mkdir actions-runner && cd actions-runner
+            curl -o actions-runner-linux-x64-2.317.0.tar.gz -L https://github.com/actions/runner/releases/download/v2.317.0/actions-runner-linux-x64-2.317.0.tar.gz
+            tar xzf ./actions-runner-linux-x64-2.317.0.tar.gz
+
+            ./config.sh --url https://github.com/${{ github.repository_owner }}/${{ github.event.repository.name }} --token ${{ env.REGISTRATION_TOKEN }} --unattended --labels self-hosted,linux,aws
+            nohup ./run.sh > nohup.out 2> nohup.err < /dev/null &
+          EOF
+
+  verify-dashboards-alerts:
+    name: Verify Dashboards and Alerts OK
+    needs: deploy-register-self-runner
+    runs-on: self-hosted
+    defaults: 
+      run:
+        shell: bash -ieo pipefail {0}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run Quickstart
+        run: |
+          sudo chown $(whoami) .
+          kind delete clusters kuadrant-local
+          kind delete clusters kuadrant-local
```

Comment:
typo on duplicate cmd?

Reply:
I found that running the command just once often proceeded in the container environment unable to delete, proceeding with an error. By performing this action twice, even if it fails the first time, it can delete the second time. I'm unsure as to why this behavior exists... Even if it succeeds the first time, it doesn't throw an error and proceeds. :)